src/udev/udev-worker.h

```diff
@@ -40,6 +40,7 @@ typedef enum EventResult {
         EVENT_RESULT_NERRNO_MAX       = -1,
         EVENT_RESULT_SUCCESS          = 0,
         EVENT_RESULT_EXIT_STATUS_BASE = 0,
+        EVENT_RESULT_EXIT_STATUS_BASE_START = 1, 
```

Comment:
This doesn't seem generally useful... If it doesn't matter either way, the status quo seems more straightforward,

Reply:
Removed the changes

Reply:
Added changes again due to build failure but changed the name to EVENT_RESULT_EXIT_STATUS_BASE_PLUS_ONE